### ISSUE TITLE
Private channel 

### ISSUE THREAD

**Issue by servantthought** [niekmussche/Discord-MCPE#36]
I have the channels set to private but it's not  coming through is there a way to setup a discord rank to allow it to go through... my ingame says messege was sent...

**Comment by NiekertDev** [comment]
Add me as @Niekert#9833

**Comment by NiekertDev** [comment]
On discord 😏

**Comment by ritokun** [comment]
Can I change the icon of BOT?

**Comment by NiekertDev** [comment]
Yes, on the same place where you edit webhooks you can select an icon

**Comment by ritokun** [comment]
私はDiscordから送られた機能をMaincraftに表示して追加して欲しい！Bukkitやスピゴットで言及されたDiscordSRV

**Comment by ritokun** [comment]
I want the function sent from Discord and displayed in Maincraft to be added! DiscordSRV mentioned in Bukkit or spigot

**Comment by servantthought** [comment]
@niekert#9833 didnt show up on discord

**Comment by NiekertDev** [comment]
You did type it correct? @ Niekert # 9833

**Comment by Aericio** [comment]
It's a webhook, so there is no way to apply a rank to it. It should still be able to send to a private channel though. I'm not necessarily sure what debug does, but could you try to enable that and try again?
